app/src/main/res/layout/fragment_view_conteiner_day_shedule.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why FrameLayout?

Reply:
For this task, thinking, it right thing.